.github/workflows/ci.yml

```diff
@@ -70,5 +70,7 @@ jobs:
         run: tar -xzvf maven-repo.tgz -C ~
       - name: Build testsuite
         run: mvn clean install -B -DskipTests -f testsuite/pom.xml
+      - name: Run base tests - role map provider
```

Comment:
This step is intentionally left separated from others for now. The reason is to isolate map role provider for now. I plan to change that once the PR is ready for review.